$MirOS: ports/emulators/qemu/patches/patch-slirp_slirp_c,v 1.2 2005/05/03 19:30:05 tg Exp $
--- slirp/slirp.c.orig	Wed Apr 27 20:51:43 2005
+++ slirp/slirp.c	Tue May  3 18:04:49 2005
@@ -80,7 +80,7 @@ static int get_dns_addr(struct in_addr *
 static int get_dns_addr(struct in_addr *pdns_addr)
 {
     char buff[512];
-    char buff2[256];
+    char buff2[257];
     FILE *f;
     int found = 0;
     struct in_addr tmp_addr;
